src/rdbms-connect/azext_rdbms_connect/_client_factory.py

```diff
@@ -0,0 +1,159 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+
+RM_URI_OVERRIDE = 'AZURE_CLI_RDBMS_RM_URI'
+SUB_ID_OVERRIDE = 'AZURE_CLI_RDBMS_SUB_ID'
+CLIENT_ID = 'AZURE_CLIENT_ID'
+TENANT_ID = 'AZURE_TENANT_ID'
+CLIENT_SECRET = 'AZURE_CLIENT_SECRET'
+
+
+def get_mariadb_management_client(cli_ctx, **_):
+    from os import getenv
+    from azure.mgmt.rdbms.mariadb import MariaDBManagementClient
+
+    # Allow overriding resource manager URI using environment variable
+    # for testing purposes. Subscription id is also determined by environment
+    # variable.
+    rm_uri_override = getenv(RM_URI_OVERRIDE)
```

Comment:
May I know the reason for overriding the resource manager URI?
If you want to test for different cloud settings, please try `az cloud set`.

Reply:
Fixing in latest iteration.